hw/xbox/xid-sdl.c

```diff
@@ -194,21 +194,22 @@ static const XIDDesc desc_xid_xbox_gamepad = {
 
 #define BUTTON_MASK(button) (1 << ((button) - GAMEPAD_DPAD_UP))
 
+SDL_HapticEffect effect;
```

Comment:
This is a hacky hacky bad hack. Shared memory across all plugged in devices. Should probably be within `USBXIDState` so each controller maintains it's own force feedback event.

Reply:
Also, if we keep it, it should be guarded by `#ifdef FORCE_FEEDBACK `. But we might want to get rid of the macro anyway.